tests/test_examples.py

```diff
@@ -131,19 +78,17 @@ def test_from_and_to_json_roundtrip(source, filename) -> None:
         raise AssertionError(msg) from err
 
 
-@pytest.mark.filterwarnings(
-    "ignore:DataFrameGroupBy.apply.*:DeprecationWarning",
-)
-@pytest.mark.parametrize(("source", "filename"), distributed_examples, ids=id_func)
-@pytest.mark.skipif(
-    not VL_CONVERT_AVAILABLE,
-    reason="vl_convert not importable; cannot run mimebundle tests",
-)
-def test_render_examples_to_png(source, filename) -> None:
+@slow
+@ignore_DataFrameGroupBy
+@distributed_examples
+@skip_requires_vl_convert
+def test_render_examples_to_png(source: Any, filename: str) -> None:
     chart = eval_block(source)
     if chart is None:
         msg = f"Example file {filename} should define chart in its final statement."
         raise ValueError(msg)
     out = io.BytesIO()
     chart.save(out, format="png", engine="vl-convert")
-    assert out.getvalue().startswith(b"\x89PNG")
+    buf = out.getbuffer()
+    prefix = buf[:4].tobytes()
+    assert prefix == b"\x89PNG"
```

Comment:
Was an attempt to speed up these tests that were identified as slow, but I don't think it made an observable improvement.

The thinking was that it avoids copying the full contents of the buffer, since all we need are the first four bytes.

See [BytesIO.getbuffer](https://docs.python.org/3/library/io.html#io.BytesIO.getbuffer)